src/main/java/ru/sbtqa/tag/parsers/JsonParser.java

```diff
@@ -23,10 +26,19 @@ public Object call(ParserItem item) {
         Object result = "";
         try {
             result = read(item.getSource(), item.getPath());
+            if (Props.get("jsonparser.toJSONString") != null
+                    && Boolean.valueOf(Props.get("jsonparser.toJSONString")) == true) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1125)